src/components/badge/badge.stories.mdx

```diff
@@ -70,13 +70,30 @@ If the counter is less than 1 badge will not show itself
 <Canvas>
   <Story name="with counter 0">
     <div style={{ margin: "40px" }}>
-      <Badge counter={0}>
+      <Badge counter={0} onClick={()=>{}}>
         <Button buttonType="tertiary">Filter</Button>
       </Badge>
     </div>
   </Story>
 </Canvas>
 
+### Display Only
+
+To acheive a `display only` version of this component, you can simply not pass an `onClick` function to the component. This means that instead of rendering as a `button` element, 
+the component will render as a `span`.
+
+<Canvas>
+  <Story name="display only">
+    <div style={{ margin: "40px" }}>
```

Comment:
Considering that we have [this PR](https://github.com/Sage/carbon/pull/5869) open I'd avoid adding new instances of `style` prop.

Reply:
Completely forgot about that PR. I'll swap the `div` out for `box`.